src/main/resources/application-migration-local.yml

```diff
@@ -0,0 +1,29 @@
+localstack:
```

Comment:
This isn't required - if you spin up with `local` and `migration` then you will get this from the other profile.

Reply:
Removed